blackbox-exporter/Dockerfile

```
@@ -13,9 +13,47 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM docker.io/prom/blackbox-exporter:v0.19.0
+ARG GO_VERSION=1.15
+
+FROM golang:${GO_VERSION}-alpine3.12 AS build
+
+ENV DISTRIBUTION_DIR /go/src/github.com/prometheus/blackbox_exporter
+ENV BUILDTAGS include_oss include_gcs
+
+ARG GOOS=linux
+ARG GOARCH=amd64
+ARG GOARM=6
+ARG VERSION
+ARG REVISION
+
+RUN set -ex \
+    && apk add --no-cache make git curl
+
+WORKDIR $DISTRIBUTION_DIR
+
+RUN git clone https://github.com/prometheus/blackbox_exporter $DISTRIBUTION_DIR && git checkout v0.19.0
+
+RUN CGO_ENABLED=0 make build
+
+FROM quay.io/astronomer/ap-base:3.13.5
```

Can we go to ap-base:3.14.0?

---

AFAIK it doesn't exists

---

ap-base 3.14.0 does exist
```suggestion
FROM quay.io/astronomer/ap-base:3.14.0
```